Subject: emu10k1: guard struct snd_emu10k1 kABI changes
From: Jiri Slaby <jslaby@suse.cz>
Patch-mainline: never

struct snd_emu10k1 shall be allocated by snd_emu10k1_create, so moving
delay_pcm_irq to the end of the structure is fine.

Introduced by 359789e0a8fb95dca2cf755e16ba8af1a20d41cf.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/sound/emu10k1.h |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/include/sound/emu10k1.h
+++ b/include/sound/emu10k1.h
@@ -1707,7 +1707,6 @@ struct snd_emu10k1 {
 	unsigned int card_type;			/* EMU10K1_CARD_* */
 	unsigned int ecard_ctrl;		/* ecard control bits */
 	unsigned long dma_mask;			/* PCI DMA mask */
-	unsigned int delay_pcm_irq;		/* in samples */
 	int max_cache_pages;			/* max memory size / PAGE_SIZE */
 	struct snd_dma_buffer silent_page;	/* silent page */
 	struct snd_dma_buffer ptb_pages;	/* page table pages */
@@ -1795,6 +1794,9 @@ struct snd_emu10k1 {
 	unsigned int saved_a_iocfg, saved_hcfg;
 #endif
 
+#ifndef __GENKSYMS__
+	unsigned int delay_pcm_irq;		/* in samples */
+#endif
 };
 
 int snd_emu10k1_create(struct snd_card *card,
